gui-libs/libdecor/libdecor-9999.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 2023-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson
+
+DESCRIPTION="A client-side decorations library for Wayland clients"
+HOMEPAGE="https://gitlab.freedesktop.org/libdecor/libdecor"
+if [[ ${PV} == "9999" ]]; then
+	EGIT_REPO_URI="https://gitlab.freedesktop.org/libdecor/libdecor.git"
+	inherit git-r3
+else
+	SRC_URI="https://gitlab.freedesktop.org/libdecor/libdecor/-/archive/${PV}/${P}.tar.bz2"
+	KEYWORDS="~amd64"
+fi
+LICENSE="MIT"
+SLOT="0"
+IUSE="+dbus +gtk examples"
+
+DEPEND="
+	>=dev-libs/wayland-1.18
+	>=dev-libs/wayland-protocols-1.15
+	x11-libs/pango
+	x11-libs/cairo
+	dbus? ( sys-apps/dbus )
+	gtk? ( x11-libs/gtk+:3 )
+	examples? (
+		virtual/opengl
+		media-libs/mesa[opengl(+)]
```

Comment:
These two should be replaced with `media-libs/libglvnd`.

Reply:
Why replace `virtual/opengl` and `media-libs/mesa` with that dependencies?

Here is the dependencies for libdecor:
meson >= 0.47
ninja
wayland-client >= 1.18
wayland-protocols >= 1.15
wayland-cursor
cairo
pangocairo
dbus-1
egl (to build EGL example)
opengl
xkbcommon (to build cairo demo)
gtk+-3.0 (for GTK plugin)

Reply:
The demo links against these libraries:

```
$ qfile /usr/lib64/libOpenGL.so /usr/lib64/libEGL.so
media-libs/libglvnd: /usr/lib64/libOpenGL.so
media-libs/libglvnd: /usr/lib64/libEGL.so
```

virtual/opengl is considered legacy and Mesa is not the only provider of these. libglvnd is designed to switch between providers.

Reply:
I was thinking mesa was the only provided and virtual/opengl was just for pkgconfig know OpenGL is install